modin/conftest.py

```diff
@@ -129,6 +129,18 @@ def simulate_cloud(request):
     if mode == "off":
         yield
         return
+    if (
+        request.config.getoption("usepdb")
+        and request.config.getoption("capture") != "no"
```

Comment:
Should we check the presence of '-s' as well?

Reply:
it's `capture` option 😉 checked at exactly the line you left the comment